src/commands/SetClipCommand.cpp

```diff
@@ -85,34 +85,34 @@ void SetClipCommand::PopulateOrExchange(ShuttleGui & S)
    S.EndMultiColumn();
 }
 
-bool SetClipCommand::ApplyInner( const CommandContext &, Track * t )
+bool SetClipCommand::Apply(const CommandContext& context)
 {
-   // if no 'At' is specified, then any clip in any selected track will be set.
-   t->TypeSwitch([&](WaveTrack &waveTrack) {
-      WaveClipPointers ptrs( waveTrack.SortedClipArray());
```

Comment:
Not sorting clips any more.  What might that change?  Nothing, I see.  No break was in the loop.  OK